dnaseq_slurm_api.sh

```diff
@@ -0,0 +1,37 @@
+# sample commands to be called by stager slurm.py with variable substitution
+# assumes crg2 is installed in home directory of user
+# in production, crg2 is located in /home/ccmmarvin
+# {family}, {analysis_id}, {participant}, {linked_files} are passed to the slurm API by slurm.py
+
+SF=~/crg2/Snakefile; 
+CP="/home/slurm/conda_envs/crg2-conda";
+SLURM=~/crg2/slurm_profile;
+
+source /storage/modules/anaconda/2020.11/etc/profile.d/conda.sh
+conda activate snakemake
+
+#temporary filepath
+filepath=/storage/data/test_crg2_automation
+analysis_dir=${filepath}/{analysis_id}/{family}
+
+if [ ! -d $analysis_dir ];then
+    mkdir  -p $analysis_dir
+else
+    echo "Analysis directory $analysis_dir exists, exiting"
+    exit 1
+fi
```

Comment:
So one consequence of this is that Stager should only attempt to run trios or single-family analyses automatically for now?

Reply:
For now. crg2 expects a 'family' identifier; I suppose in the future we could we could exclude the family codename from the filepath, and then use some logic either on the Stager side or crg2 side to define the family codename as the family codename if all individuals come from the same family, or something generic like 'cohort' if the set of family_codenames is greater than 1.